packages/language-pack/en.json

```diff
@@ -1931,7 +1931,7 @@
 					"DISABLE_TITLE": "Disable User",
 					"ENABLE_DESC": "Are you sure you want to enable selected user below?",
 					"ENABLE_TITLE": "Enable User",
-					"INVITE_TITLE": "Invite Users to Workspace-Name",
+					"INVITE_TITLE": "Invite Users to {workspace_name}",
```

Comment:
question: `workspace_name` variable is added. 
Does the developer know about it?

Reply:
yes! I know, thx!👍🏻